Dockerfile

```diff
@@ -1,17 +1,20 @@
 
+FROM composer AS builder
+
+COPY ./ /app
+RUN composer install
+
 # Running this container will start a language server that listens for TCP connections on port 2088
 # Every connection will be run in a forked child process
```

Comment:
Can you keep this comment at the top?

Reply:
Yes of course :)

Reply:
Moved the comments back to the top. I also replaced the deprecated maintainer with the appropriate "maintainer" label.